src/vendor/bullet.js

```diff
@@ -8,6 +8,7 @@ export default function bullet () {
     markers = bulletMarkers,
     measures = bulletMeasures,
     measureHeight = bulletMeasureHeight,
+    rangeMaxx = getRangeMax,
```

Comment:
`rangeMaxx` is off, was `maxRange` taken? or maybe `originalRangeMax` or `rawRangeMax`. Whatever feels more fitting.